gtest/Cargo.toml

```diff
@@ -14,7 +14,8 @@ codec = { package = "parity-scale-codec", version = "3.1.2", features = ["derive
 logger = { package = "log", version = "0.4.17" }
 hex = "0.4.3"
 colored = "2.0.0"
+derive_more = "0.99.17"
```

Comment:
there is features to optimise imports
like 
`default_features=false, features=["add", "display"]`

Reply:
Nice idea! We can propagate it at all places we use `derive_more` but I've fixed it only in my newly introduced code.